sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileProperties.java

```diff
@@ -199,4 +243,81 @@ public CopyStatusType copyStatus() {
     public Boolean isServerEncrypted() {
         return isServerEncrypted;
     }
+
+    /**
+     * @return The SMB properties of the file.
+     */
+    public FileSmbProperties smbProperties() {
+        return smbProperties;
+    }
+
+    /**
+     * Sets the content type.
+     * @param contentType The content type.
+     * @return the updated FileProperties object.
+     */
+    public FileProperties contentType(String contentType) {
+        this.contentType = contentType;
+        return this;
+    }
+
+    /**
+     * Sets the content md5.
+     * @param contentMD5 The content md5.
+     * @return the updated FileProperties object.
+     */
+    public FileProperties contentMD5(byte[] contentMD5) {
+        this.contentMD5 = contentMD5;
```

Comment:
`ImplUtils.clone` the byte array as you do in the constructor, this is causing a Spotbugs issue